docs/relations_guide.md

```diff
@@ -137,6 +137,14 @@ Use this relationship where only part of the process occurs in the cell type.
 
 e.g.  'retinal bipolar neuron' 'capable of part of' some 'visual perception'
 
+### Recording neurotransmitter for neurons
+
+To record which neurotransmitter a neuron releases, use a [**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215) relation to link the neuron to a  to a subclass of GO [neurotransmitter secretion](http://purl.obolibrary.org/obo/GO_0007269).  This should be sufficient for autoclassification.
+
+[**'capable of'**](http://purl.obolibrary.org/obo/RO_0002215)
```

Comment:
It looks like it is missing an example here. e.g.
'GABAergic neuron' 'capable of' some 'gamma-aminobutyric acid secretion, neurotransmission'